src/main/java/Duke.java

```diff
@@ -1,10 +1,86 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String logo = "_________________________________________________\n";
+
+        System.out.println(logo + "Hello! I'm Duke\n" + "Whatchu want\n" + logo);
+
+        Scanner in = new Scanner(System.in);
+
+        Task[] Task = new Task[100];
+        String word;
+        String date = "";
+        int Count = 0;
+
+        do {
+            word = in.nextLine();
+            String[] words = word.split(" "); // stores string of multiple words in a string Array
+
+            if (isDone(words[0])) { //complete Tasks
+                int i = Integer.parseInt(words[1]) - 1; // gets Task number
+                Task[i].setDone();
+                System.out.println(logo + "Nice! I've marked this task as done:\n" + Task[i] + "\n" + logo);
+            }
+
+            else if (!isBye(word) && !isList(word)) { //Task words
+                if (isTodo(words[0])) { //Todo task
+                    Task[Count] = new Todo(word);
+                    Count++;
+                }
+                else if (isDeadline(words[0])) { //Deadline task
+                    date = word.substring(word.lastIndexOf("/") + 1);
+                    Task[Count] = new Deadline(word, date);
+                    Count++;
+                }
+
+                else if (isEvent(words[0])) { //Event task
+                    date = word.substring(word.lastIndexOf("/") + 1);
+                    Task[Count] = new Event(word, date);
```

Comment:
Date is not a good variable name since this variable when used in the Event class can contain location data too.